doc/discovery.md

```diff
@@ -171,6 +171,110 @@ linux-pc:# avahi-browse -ar
 ^C
 linux-pc:#
 ```
+
+Additionally, *avahi-resolve-host-name* can be used to verify domain
+name mappings for IP addresses. By default, it translates from IPv4
+addresses. This function allows users to confirm that addresses are
+mapped correctly.
+
+```
+linux-pc:# avahi-resolve-host-name infix-00-00-00.local
+infix-00-00-00.local	10.0.1.1
+linux-pc:#
+```
+
+Resolved addresses can also be used for tasks like pinging or
+establishing an SSH connection to the device, as demonstrated below:
+
+```
+linux-pc:# ping infix-00-00-00.local -c 3
+PING infix-00-00-00.local (10.0.1.1) 56(84) bytes of data.
+64 bytes from 10.0.1.1: icmp_seq=1 ttl=64 time=0.852 ms
+64 bytes from 10.0.1.1: icmp_seq=2 ttl=64 time=1.12 ms
+64 bytes from 10.0.1.1: icmp_seq=3 ttl=64 time=1.35 ms
+
+--- infix-00-00-00.local ping statistics ---
+3 packets transmitted, 3 received, 0% packet loss, time 2003ms
+rtt min/avg/max/mdev = 0.852/1.105/1.348/0.202 ms
+
+linux-pc:# ssh admin@infix-00-00-00.local
+(admin@infix-00-00-00.local) Password: 
+.-------.
+|  . .  | Infix -- a Network Operating System
+|-. v .-| https://kernelkit.org
+'-'---'-
+
+Run the command 'cli' for interactive OAM
+
+linux-pc:#
+```
+
+To disable mDNS/mDNS-SD, type the commands:
+```
+admin@infix-00-00-00:/> configure 
+admin@infix-00-00-00:/config/> set mdns enabled false 
+admin@infix-00-00-00:/config/> leave
+```
+
+#### Short Hostname 
+
+With the help of the mdns-alias, mDNS functionality allows the device
+to be accessible as *infix.local* rather than its full hostname 
+(e.g., infix-00-00-00.local). This alias works seamlessly on a network
+with a single Infix device. However, if multiple Infix devices are
+present (e.g., infix-00-00-00.local, infix-01-01-01.local), each device
+requires a unique hostname to avoid conflicts, as infix.local would not
+uniquely identify multiple devices. The shortened names can also be
+used for actions like pinging or establishing an SSH connection:
+
```

Comment:
This needs to be simplified.   The `mdns-alias` program is a background helper process we shouldn't mention to the user, if Avahi adds native support for this in the future we will replace `mdns-alias` with that.

Also, the `infix.local` is not shortform of `infix-xx-yy-zz.local`, it's hard-coded to the `DEFAULT_HOSTNAME` from the file `/etc/os-release` + `.local`.  The file `/etc/os-release` in turn is unique for each build, for a generic Infix build `DEFAULT_HOSTNAME=infix`, or `DEFAULT_HOSTNAME=ix` for the minimal builds.  Customer builds will have other defaults.

We need to explain that and the fact that only one device can "own" the alias.  For mDNS it's the first device on the LAN that retains the privilege.  All other devices joining in on the LAN will realize they are not allowed to advertise `infix.local`.

Reply:
Fixing parts of this. Hope it is good enough for now, but comments are of course welcome!